src/main/java/net/minecraftforge/event/level/ChunkEvent.java

```diff
@@ -91,10 +84,26 @@ public boolean isNewChunk()
      * <br>
      * This event is fired on the {@link MinecraftForge#EVENT_BUS}.<br>
      **/
-    public static class Unload extends ChunkEvent
-    {
-        public Unload(ChunkAccess chunk)
-        {
+    public static class Unload extends ChunkEvent {
+        public Unload(ChunkAccess chunk) {
+            super(chunk);
+        }
+    }
+    
+    /**
+     * ChunkEvent.LightingCalculated is fired when MinecraftForge flags that lighting is correct in a chunk.<br>
+     * This event is fired during light propagation in ThreadedLightEngine.CompletableFuture(), specifically upon setting 
+     * the ChunkAccess isLightCorrect to true.<br>
+     *
+     * ThreadedLevelLightEngine.CompletableFuture(). <br>
+     * This event is not {@link Cancelable}.<br>
+     * <br>
+     * This event does not have a result. {@link HasResult} <br>
+     * <br>
+     * This event is fired on the {@link MinecraftForge#EVENT_BUS}.<br>
+     **/
+    public static class LightingCalculated extends ChunkEvent {
```

Comment:
Overall looks good, one minor thing from me beyond Jonathan's comments.

Please mark the class as `final` if you don't intentionally support developers extending this event or don't foresee any reason for them doing so. A related PR aims to mark many existing events in Forge as final: https://github.com/MinecraftForge/MinecraftForge/pull/10515
```suggestion
    public static final class LightingCalculated extends ChunkEvent {
```

Reply:
Completed.